core/src/main/java/org/apache/accumulo/core/conf/Property.java

```diff
@@ -228,6 +228,15 @@ public enum Property {
       "2.1.0"),
   GENERAL_RPC_TIMEOUT("general.rpc.timeout", "120s", PropertyType.TIMEDURATION,
       "Time to wait on I/O for simple, short RPC calls", "1.3.5"),
+  GENERAL_FILENAME_PREFIX("general.filename.", null, PropertyType.PREFIX,
```

Comment:
I don't believe the prefix property is needed. This is only necessary when custom plugins can be created that will expect properties to be set that are not defined in Property.java.

Reply:
I wanted to make this a valid zoo property so that it could be configured via the config shell command. Do you have a recommended way to do that without the new prefix?